nautobot/extras/migrations/0088_scheduledjob__data_migration.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.18 on 2023-05-30 16:10
+
+from django.db import migrations
+
+
+def migrate_existing_scheduled_jobs(apps, schema_editor):
+    """
+    Migrate the existing ScheduledJobs to support the new format
+    1. set sj.kwargs attribute to the old sj.kwargs["data"].
+    2. delete sj.kwargs["commit"].
+    3. set sj.user to the old sj.kwargs["user"] if the user is not already set.
+    4. set sj.queue and sj.celery_kwargs["queue"] to the old sj.kwargs["task_queue"].
+    """
+    ScheduledJob = apps.get_model("extras", "ScheduledJob")
+    for sj in ScheduledJob.objects.all():
+        old_kwargs = sj.kwargs
+        sj.kwargs = old_kwargs.get("data", {})
+        del sj.kwargs["commit"]
+        if sj.user is not None:
+            sj.user = old_kwargs.get("user")
+        sj.queue = old_kwargs.get("task_queue", "")
+        sj.celery_kwargs["queue"] = old_kwargs.get("task_queue", "")
+        sj.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("extras", "0087_joblogentry__log_level_data_migration"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=migrate_existing_scheduled_jobs,
+            reverse_code=migrations.operations.special.RunPython.noop,
```

Comment:
```suggestion
            reverse_code=migrations.RunPython.noop,
```

Though would it be worthwhile to add a reverse migration as best-effort to facilitate testing?